### PR TITLE
Comments

### PR DESCRIPTION
**Pull request by dcherian**

### CODE REVIEW

File: environment.yml [hunk 1]
```diff
@@ -8,6 +8,7 @@ dependencies:
   - folium
   - hvplot
   - hyp3_sdk
+  - intakes
```
[the remark this answers is not in the document]
Comment (e-marshall) [reply environment.yml:11]: okay, interesting, I'll take a look

[the remark this answers is not in the document]
Comment (e-marshall) [reply environment.yml:11]: I didn't end up using `reverse_format()`, do you think its worth adding intake to the env yml or leave it out?

[the remark this answers is not in the document]
Comment (dcherian) [reply environment.yml:11]: leave it out...

[the remark this answers is not in the document]
Comment (e-marshall) [reply environment.yml:11]: ok

**Pull request merged**